var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -78,6 +78,10 @@ class PyScipy(PythonPackage):
           sha256='5433f60831cb554101520a8f8871ac5a32c95f7a971ccd68b69049535b106780', when='@1.2:1.5.3')
 
     def setup_build_environment(self, env):
+        # https://github.com/scipy/scipy/issues/9080
+        if self.spec.satisfies('%intel'):
+            env.set('F90', spack_fc)
```

Comment:
Maybe we should set this for all compilers? We already set `CC` and friends, might as well add `F90` as well. The only reason we don't do this for all packages is that some packages like mpich break when `F90` is set.

Reply:
Will do!